cms/static/cms/sass/components/_modal.scss

```diff
@@ -193,7 +193,7 @@
 
 .cms-modal-resize {
```

Comment:
I think the resize cursor should also be flipped (and patched in `_rtl_patch.scss`):

```css
    &:dir(rtl) {
        cursor: ne-resize;  // cursor pointing north-east
    }
```